internal/routes/all_routes.go

```diff
@@ -72,10 +72,9 @@ func MountAPI(r *chi.Mux) {
 				// TODO DEPRECATED: move this to outside of /sources (see below)
 				r.Get("/instance_types", s.ListInstanceTypes)
 
-				// TODO DEPRECATED: replaced with upload_info
```

Comment:
Don’t remove deprecation warning, I believe this was for account_identity endpoint.